### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by veganaize**
Addresses issue #89 

### 👨‍💻 Changes proposed

Adds a CODE_OF_CODUCT

_I was just reading about how to add a code of conduct and it funneled me all the way to this pull request.  So I'm just offering it to save someone else the time!_

### CODE REVIEW

**Comment by ykdojo** [comment]
Thank you. Did you copy it from somewhere else?

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+This Code of Conduct applies within all community spaces, and also applies when
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement.
```
Comment (veganaize) [inline CODE_OF_CONDUCT.md:62]: The original template had a blank spot, at the end of this line, for an email address or at-mention (I suppose).  I removed the word "at" and the blank area.

**Comment by veganaize** [comment]
> Thank you. Did you copy it from somewhere else?

Yes.  It's the default template that Github provides.  To my understanding, Github requires the use of a template in order to satisfy the [Community Standards checklist](https://github.com/ykdojo/defaang/community).

**Comment by ykdojo** [comment]
@veganaize I'd rather not have it if it's copied from a template. It'd be better to write it from scratch.

**Comment by veganaize** [comment]
> @veganaize I'd rather not have it if it's copied from a template. It'd be better to write it from scratch.

_Okay, no worries @ykdojo !_

**Comment by ykdojo** [comment]
@veganaize cool! If you want, you can commit an empty code of conduct (or maybe just one line) and have people suggest changes.

**Comment by veganaize** [comment]
@ykdojo Okay, reopening PR with an empty / one-liner code of conduct which can be previewed [here](https://github.com/veganaize/defaang/blob/patch-1/CODE_OF_CONDUCT.md).

**Comment by veganaize** [comment]
@ykdojo Okay, reopening PR with an empty / one-liner code of conduct which can be previewed [here](https://github.com/veganaize/defaang/blob/patch-1/CODE_OF_CONDUCT.md).

**Review by ykdojo — APPROVED** [review]

**Comment by ykdojo** [comment]
Looks good! I'll wait for another reviewer.

**Comment by veganaize** [comment]
@subhoghoshX Would you mind reviewing this, at your convenience?

**Comment by subhoghoshX** [comment]
Thanks for the effort @veganaize. LGTM.

Btw @ykdojo, I don't think there's anything wrong with using the default template provided by github. I checked React, Svelte, NextJS, and Gatsby they all used the template and made modifications to it. I think we can use it and make enough changes to make it our own. As I don't think we have a professional copywriter on our team :laughing:

**Review by subhoghoshX — APPROVED** [review]

**Comment by ykdojo** [comment]
@subhoghoshX the problem with the template is that:

1. It's too long. I can't really even understand it without spending a lot of time on it.
2. It's not ours. It's a rule that someone else created.

I think those projects simply copy it just because other projects do, too.

I was comfortable using a template for the license because it's:

1. A legal document
2. Short

Neither of which is the code of conduct template.

For not having a copywriter - I can do that until we have another person who can do it, too.

**Comment by subhoghoshX** [comment]
> @subhoghoshX the problem with the template is that:
> 
> 1. It's too long. I can't really even understand it without spending a lot of time on it.
> 2. It's not ours. It's a rule that someone else created.
> 
> I think those projects simply copy it just because other projects do, too.
> 
> I was comfortable using a template for the license because it's:
> 
> 1. A legal document
> 2. Short
> 
> Neither of which is the code of conduct template.
> 
> For not having a copywriter - I can do that until we have another person who can do it, too.

Understood. Thanks for the explanation.

**Pull request merged**